From 39b97cc0138417141b245179fc8555bbb365e879 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Fri, 8 Jul 2011 12:24:16 +1000
Subject: Always set the type name.

The server assigns "UNKNOWN", override it with something more specific.

Introduced in f85c4b580c074f7054eac98753d1f4e91f08305e.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>

diff --git a/src/acecad.c b/src/acecad.c
index 604fe91..2b6aa45 100644
--- src/acecad.c
+++ src/acecad.c
@@ -346,6 +346,7 @@ AceCadPreInit(InputDriverPtr drv, InputInfoPtr local, int flags)
     local->private = priv;
 
     local->device_control = DeviceControl;
+    local->type_name = XI_TABLET;
 
     priv->acecadInc = xf86SetIntOption(local->options, "Increment", 0 );
 
-- 
cgit v0.10.2

